daemon/kmd/wallet/driver/ledger.go

```diff
@@ -209,7 +209,25 @@ func (lw *LedgerWallet) SignTransaction(tx transactions.Transaction, pw []byte)
 
 // MultisigSignTransaction implements the Wallet interface.
 func (lw *LedgerWallet) MultisigSignTransaction(tx transactions.Transaction, pk crypto.PublicKey, partial crypto.MultisigSig, pw []byte) (crypto.MultisigSig, error) {
-	return crypto.MultisigSig{}, errNotSupported
+
+	isValidKey := false
+	for i := 0; i < len(partial.Subsigs); i++ {
+		subsig := &partial.Subsigs[i]
+		if subsig.Key == pk {
+			isValidKey = true
+			sig, err := lw.signTransactionHelper(tx)
```

Comment:
If the user's key appears N times in the multisig array, this code will ask the user to sign the transaction N times on their ledger.  It would be good to avoid that, and ask just once.